README.md

```diff
@@ -0,0 +1,438 @@
+
+A good way to handle maintaining your own private copy if you use Subversion for your project is to use the svn:externals mechanism to automatically reference a specific revision of PoDoFo from podofo svn, or a particular tag . This permits you to control when you update to a new version and lets you make sure your code is updated for any API changes, without forcing users to go and find a specific version of PoDoFo. Alternately, you can just check the tree into your own revision control system and do periodic code drops. Have a look at PoDoFoBrowser for an example of the svn:externals approach.
+
+If you statically link to podofo, that'll prevent any unexpected problems with linker paths. Another option on most platforms is to use rpath linking to ensure that a specific copy of the podofo library is preferred by the linker. CMake takes care of this automatically if you're using it for your project.
+
+When PoDoFo reaches 1.x versions it is expected that API changes will be less frequent. At this point, though, they're an unavoidable part of improving and completing the library.
+
+If you are using CMake in your project and you choose to bundle a copy of the PoDoFo sources, with svn:externals or otherwise, you can quite easily build a private copy of PoDoFo as part of your app's normal build process. Have a look at FindLIBPODOFO.cmake from PoDoFoBrowser to see how to do that, or just copy that file.
+
+### Compiling and Linking Against PoDoFo
+
+PoDoFo's headers are designed to avoid exposing its library dependencies, so you do not need the headers for Freetype etc on the include path. However, you MUST link your application to podofo's required libraries on most platforms, eg on UNIX:
+
+    -lpodofo -lfreetype -lfontconfig -ljpeg -lz
+
+is generally appropriate. Win32 users must also ensure that they link to `gdi32` and `ws_win32`.
+
+### Handling Exceptions
+
+When working with PoDoFo it is important to remember that the library makes use of exceptions. Your code must be built with exception support and must be exception safe in areas that call PoDoFo. gcc users may not build with -fno-exceptions. If you have non-exception-safe code or code (such as pure C code) that cannot propagate exceptions your podofo-using code must catch all exceptions before they propagate to such unsafe sections. PoDoFo will throw a PdfError& for PoDoFo-specific exceptions. It may also propagate STL/runtime generated exceptions like std::bad\_alloc, though it'll never throw them directly. PoDoFo cannot be built without exceptions.
+
+More general information about exception handling is in the [FAQ](FAQ.html).
+
+## Preprocessor defines used in PoDoFo
+
+Several preprocessor defines are used inside PoDoFo. This section tries to document all of them. These defines are set automatically by the CMake build system, so usually you do not have to worry about them.
+
+**Define**
+
+**Meaning**
+
+`DEBUG`
+
+Indicates a debug build of PoDoFo.
```

Comment:
The formatting isn't great. I'd suggest we at least leave out the (pseudo-)headers, and ideally would use a [definition list](https://www.markdownguide.org/extended-syntax/#definition-lists).